src/endpoints/auth/sign-up.route.ts

```diff
@@ -48,17 +51,22 @@ export const signUpRoute: RouteCustomOptions<{ Body: ISignUpOrInBodyInput }> = {
 
     await VerificationCodeEntity.create({
       userId: String(dataUser._id),
-      code: '123456',
+      code,
       expiresAt,
     }).save();
 
-    await EmailService.sendMessageToEmail(email, code, EmailEnum.REGISTRATION_LETTER);
+    try {
+      await EmailService.sendMessageToEmail(email, code, EmailEnum.REGISTRATION_LETTER);
+    } catch (err) {
+      new InternalServerError(err);
```

Comment:
The same. This is quite a specific (and possible) error, it's better to log it properly.